core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -399,6 +404,16 @@ class MapUnit {
         tile.improvementInProgress = null
     }
 
+    private fun tryProvideProductionToClosestCity()
+    {
+        val tile = getTile()
+        val closestCity = civInfo.cities.minBy { it.location.dst(tile.position) }
+        if (closestCity == null) return
+        val productionToAdd = (25 - closestCity.location.dst(tile.position) * 4).toInt()
+        if (productionToAdd > 0) closestCity.cityConstructions.addProductionPoints(productionToAdd)
+        //civInfo.addNotification("[$productionToAdd] Production points added towards current construction in [${closestCity.name}]", closestCity.location, Color.YELLOW)
```

Comment:
And yes, we should enable this notification :)

Reply:
enabled